dask_cloudprovider/aws/ecs.py

```diff
@@ -953,6 +968,16 @@ async def _start(
                 or await self._create_security_groups()
             )
 
+        self._worker_task_def_name = (
+            self._task_def_name if self._task_def_name else "dask-worker-{uuid}"
+        ).format(uuid=str(uuid.uuid4())[:5], **os.environ)
+
+        self._scheduler_task_def_name = (
+            f"{str(self._task_def_name)}-scheduler-{uuid}"
```

Comment:
This still needs fixing up the same as the worker task name

Reply:
not sure I quite understand, if we don't add `scheduler` in there somewhere, the worker and scheduler task definitions would be the same and that would cause conflicts